custom-graders/v2/README.md

```diff
@@ -0,0 +1,37 @@
+# New V2 Format Custom grader samples
+
+### What is new in V2?
+Please refer to this [doc](https://docs.google.com/UPDATE_THE_LINK) to know more about what is available in autograder v2.
+
```

Comment:
Would it be possible to embed the "Grader format requirements" section in the doc, into this README as well? It might be easier for people to review the format changes that way.

Reply:
Can also add after "Please refer to this [doc]..." 

'The "v2 format" for autograders will be come the default format starting **November 18**. If you are actively creating autograders between October 2020 and January 2021, we highly encourage you to reach out to your partner support teams for early access to the v2 infrastructure. This will ensure that you can incorporate v2 formatting into your autograder development, as soon as possible. Please check out the link above for details on gaining early access.'

Reply:
Good idea. Will do.